src/data/sample-audio-responses.json

```diff
@@ -250,6 +252,14 @@
       "name": "Distance & Temps",
       "url": "https://www.jamendo.com/album/1290/distance-and-temps"
     },
+    "alt_files": [{
+      "extension": "flac",
+      "url": "https://mp3d.jamendo.com/download/track/8561/flac"
+      },
+      {
+      "extension": "mp31",
+      "url": "https://mp3d.jamendo.com/download/track/8561/mp31"
+    }],
```

Comment:
Why use `extension` here instead of `filetype` to be consistent with the main result?

Reply:
yep, might be best to match the catalog fields, no?

https://github.com/WordPress/openverse-catalog/blob/a63fdf81755632f9c6ba83c822fd0fe8eef853a6/openverse_catalog/dags/provider_api_scripts/freesound.py#L198-L204

Reply:
This is the main reason why this PR is a draft: I wanted to discuss and finalized the naming and values for `extension` ( filetype?, human-readable-filetype?). I used `extension` for now because that is what's used in the `DownloadButton` component.
So, to make the naming consistent, I guess we could use `filetype` and `url` properties for main audio, and change the [`alt_files`](https://github.com/WordPress/openverse-catalog/blob/a63fdf81755632f9c6ba83c822fd0fe8eef853a6/openverse_catalog/dags/provider_api_scripts/freesound.py#L198-L204) to have the same properties (filetype instead of format):
```
alt_files: {
        'url': media_data.get('download'),
        'filetype': media_data.get('type'),
        'filesize': media_data.get('filesize'),
        'bit_rate': media_data.get('bitrate'),
        'sample_rate': media_data.get('samplerate'),
}

Then I think it would be easier to have a technical name in the catalog (ie. `mp31`, `mp32`), and convert it to a human-readable form on the front end. So, the front end would have a mapping to convert `mp31` coming from the API to `MP3 96kbs` for display on the button.

Reply:
I've opened a discussion about this: https://github.com/WordPress/openverse/discussions/85